CPAC/utils/utils.py

```diff
@@ -491,46 +494,6 @@ def get_operand_string(mean, std_dev):
     return str1 + " -mas %s"
 
 
-def get_roi_num_list(timeseries_file, prefix=None):
-    # extracts the ROI labels from the 3dROIstats output CSV file
-    with open(timeseries_file, "r") as f:
-        roi_file_lines = f.read().splitlines()
-
-    roi_err = (
-        "\n\n[!] The output of 3dROIstats, used in extracting the "
-        "timeseries, is either empty, or not in the expected "
-        "format.\n\nROI output file: {0}\n\nIf there are no rows "
-        "in the output file, double-check your ROI/mask selection."
-        "\n\n".format(str(timeseries_file))
-    )
-
-    for line in roi_file_lines:
-        if "Mean_" in line:
-            try:
-                roi_list = line.split(",")
-                # clear out any blank strings/non ROI labels in the list
-                roi_list = [x for x in roi_list if "Mean" in x]
-                # rename labels
-                roi_list = [
-                    x.replace("Mean", "ROI").replace(" ", "").replace("#", "")
-                    for x in roi_list
-                ]
-            except:
-                raise Exception(roi_err)
-            break
-    else:
-        raise Exception(roi_err)
-
-    if prefix:
-        temp_rois = []
-        for roi in roi_list:
-            roi = prefix + "_" + str(roi)
-            temp_rois.append(roi)
-        roi_list = temp_rois
-
-    return roi_list
```

Comment:
This function isn't used anywhere except a commented-out section so I deleted both this function and the commented-out code.